HW6/task_1_func.py

```diff
@@ -0,0 +1,16 @@
+# ===============================================  TASK 1 =============================================
+
+def exponentiation(num, st):
+    return num**st
```

Comment:
nit: add spaces before and after operations
```suggestion
    return num ** st
```